python/ray/dashboard/modules/actor/actor_head.py

```diff
@@ -136,6 +136,20 @@ def __init__(self, dashboard_head):
         self.accumulative_event_processing_s = 0
 
     async def _update_actors(self):
+        """
+        Yields actor info. First yields all actors from GCS, then subscribes to
```

Comment:
this method doesn't yield anything?

Reply:
we should make the code consistent for actor_head and node_head: e.g. both use AsyncGenerator

Reply:
yes but let's make it another PR.

Reply:
we should at least fix the comment to match the current PR